include/e4/e4.h

```diff
@@ -132,14 +66,14 @@ typedef struct _e4storage e4storage;
    failure.
 
  */
-int e4c_protect_message(uint8_t *ciphertext,
+/*int e4c_protect_message(uint8_t *ciphertext,
```

Comment:
This needs to stay here as documentation for the common function format that is implemented. I could delete it, but in pubkey mode and symkey mode this is the actual signature... it's just hidden because C is ugly.